telegram/ext/_conversationhandler.py

```diff
@@ -366,8 +365,7 @@ def __init__(
                         stacklevel=2,
                     )
 
-        if self.run_async:
-            for handler in all_handlers:
+            elif self.run_async:
```

Comment:
this should be an `if` right? Just to be safe..

Reply:
both that and timeout, yeah. Those have nothing to do with the previous ifs.